docs/examples/generatedCanonicalFields.json

```diff
@@ -0,0 +1,822 @@
+[
+  {
+    "label": "Org Name",
+    "shortCode": "Org Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Mission Statement",
+    "shortCode": "Org Mission Statement",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Website",
+    "shortCode": "Org Website",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Entity Type",
+    "shortCode": "Org Entity Type",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Registration Number: EIN / TIN",
+    "shortCode": "Org Registration Number: EIN / TIN",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Address",
+    "shortCode": "Org Address",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Phone",
+    "shortCode": "Org Phone",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Email",
+    "shortCode": "Org Email",
+    "dataType": "string"
+  },
+  {
+    "label": "Org DBA Name",
+    "shortCode": "Org DBA Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Banking Information",
+    "shortCode": "Org Banking Information",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Legal Docs",
+    "shortCode": "Org Legal Docs",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Board Members",
+    "shortCode": "Org Board Members",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Administrative Lead",
+    "shortCode": "Org Administrative Lead",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Annual Operating Budget",
+    "shortCode": "Org Annual Operating Budget",
+    "dataType": "number"
+  },
+  {
+    "label": "Org Lobbying Activities",
+    "shortCode": "Org Lobbying Activities",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Start Date",
+    "shortCode": "Org Start Date",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Geographic Areas",
+    "shortCode": "Org Geographic Areas",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Audited Financial Statements",
+    "shortCode": "Org Audited Financial Statements",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Grant Agreement Signatory",
+    "shortCode": "Org Grant Agreement Signatory",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Fiscal Year End Date",
+    "shortCode": "Org Fiscal Year End Date",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Name(s)",
+    "shortCode": "Proposal Name(s)",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Primary Contact Name",
```

Comment:
I think the `Proposal` scope could (should?) be considered the default for all canonical fields (e.g. we can remove the `Proposal` prefix).

Reply:
See https://github.com/PhilanthropyDataCommons/service/pull/103#issuecomment-1289394036